.github/workflows/certora_recovery.yml

```diff
@@ -0,0 +1,44 @@
+name: certora
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rule: ['verifyRecoveryModule.sh']
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install python
+        uses: actions/setup-python@v4
+        with: { python-version: 3.11 }
+
+      - name: Install certora cli
+        run: pip install -Iv certora-cli==6.1.3
+
+      - name: Install solc
+        run: |
+          wget https://github.com/ethereum/solidity/releases/download/v0.8.23/solc-static-linux
+          chmod +x solc-static-linux
+          sudo mv solc-static-linux /usr/local/bin/solc8.23
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Verify rule ${{ matrix.rule }}
+        run: |
+          echo "key length" ${#CERTORAKEY}
+          chmod +x ./certora/scripts/${{ matrix.rule }}
+          ./certora/scripts/${{ matrix.rule }}
```

Comment:
I'm personally not a fan of these scripts - they just call `certoraRun` with the configuration file (which already specifies all the command line arguments), and we can just use the configuration files directly like we do in `safe-locking`:

https://github.com/safe-global/safe-locking/blob/5a8c080dc20a6a15a0c390f4a8056a39c97e0087/.github/workflows/certora.yml#L44-L49

Reply:
Updated 👍🏾